app/styles/mobile.scss

```diff
@@ -69,8 +69,8 @@
   padding-top: 50px;
   text-align: center;
   background-color: $black;
-  a{
-    color:$white;
+  a:hover{
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented 4 spaces, but was indented 2 spaces<br>Opening curly brace `{` should be preceded by one space